app/helpers/notifier_helper.rb

```diff
@@ -11,6 +11,15 @@ def post_message(post, opts={})
     rendered.presence || post_page_title(post)
   end
 
+  def truncated_post_message(post)
+    if post.respond_to? :message
+      plain_text = post.message.try(:plain_text_without_markdown).presence || post_page_title(post)
+      truncate(plain_text, length: 300)
+    else
+      I18n.t "notifier.a_post_you_shared"
+    end
+  end
+
```

Comment:
This has nothing to do with the notifier, so it shouldn't be added here.

Also, as commented in https://github.com/diaspora/diaspora/pull/8035#discussion_r956791512, I would just use `post_page_title(post)` in all cases (which already is truncated), so this isn't even needed at all (also I think this should already be fixed in #8035, so you wouldn't need to do anything).

Reply:
Moved (and changed) to report_helper